### DISCUSSION TITLE
NEXT STEP

### DISCUSSION THREAD

**Discussion by aespablo** [envybros/nomads-planet#46] [Announcements]
### 2차 뚫은 기념으로... 다음 스텝 생각해보자.

1. 매치메이킹 무슨 일이 있어도 고치기
2. "MAP" 저거 써서 광장 같은 거 구현도 해보기
3. 차고 시스템 및 아이템 도입 (차 꾸미기, 차량 종류 추가)
4. 레이싱 콘텐츠 추가

이까지만 해도.. 꽤 재미있을 수 있어

블록체인은 좀 별로인 느낌.. 토큰 관련 리포지토리도 지우자

**Comment by pulppixel** [comment]
3차까지 진행했고, 결과는 장려상이래. 

이거 꾸준히 해보자 그래도

**Comment by pulppixel** [comment]
0. R&D driving system
 - Updating All
 - File System CleanUp
1. VR Contents
2. Plaza hole (for social)
3. Racing Contents

**Comment by pulppixel** [comment]
모델링 배우는 즉시, 캐릭터와 차량 전부 업데이트 될 거야
